apps/zoomcallstatus/zoom_call_status.star

```diff
@@ -0,0 +1,105 @@
+""")
+
+def get_schema():
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Text(
+                id = "email",
+                name = "Zoom User Email",
+                desc = "Your personal zoom email.",
+                icon = "user",
+            ),
+            schema.Text(
+                id = "token",
+                name = "Zoom JWT Token",
+                desc = "Please create a JWT Zoom app and paste your JWT token here.",
+                icon = "key",
+            ),
+        ],
+    )
+
+def show_warning(message):
+    return render.Root(
+        child = center_children(
+            [
+                render.WrappedText(message),
+            ],
+        ),
+    )
+
+def get_user(expected_email, token):
+    res = http.get(PAGED_URL, headers = {"Authorization": "Bearer " + token})
+    if res.status_code != 200:
+        return "Invalid token"
+    foundUser = cache.get("status")
+    if foundUser != None:
+        return foundUser
+    for item in res.json()["contacts"]:
+        if expected_email == item["email"]:
+            foundUser = item
+            break
+    if foundUser == None:
+        return "Invalid email"
+    status = foundUser["presence_status"]
+    cache.set("status", status, CACHE_TTL)
+    return status
+
+def center_children(children):
+    return render.Box(
+        render.Row(
+            expanded = True,
+            main_align = "space_evenly",
+            cross_align = "center",
+            children = children,
+        ),
+    )
+
+def offline():
+    return render.Root(
+        child = center_children(
+            [
+                render.Text("Off Air"),
+            ],
+        ),
+    )
+
+def online():
+    return render.Root(
+        child = center_children(
+            [
+                render.Text("On Air"),
+                render.Image(src = RECORD_ICON, width = 20, height = 20),
+            ],
+        ),
+    )
+
+def main(config):
+    user_email = config.str("email")
+    token = config.str("token")
+    if user_email == None or token == None:
```

Comment:
It'd be great if the app could show an illustrative default graphic when no config is present. That way users can get a sense of what the app does and what it looks like, before they make the leap to generate JWTs and set it up for real.